crowdsourcing/models.py

```diff
@@ -442,6 +442,7 @@ class UserPreferences(models.Model):
     currency = models.ForeignKey(Currency)
     login_alerts = models.SmallIntegerField(default=0)
     last_updated = models.DateTimeField(auto_now_add=False, auto_now=True)
+    auto_skip = models.BooleanField(default=True)
```

Comment:
probably auto_accept or accept_next is the right property here; should match the functionality